cpp/bibelwissenschaft/extract_names.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [[ $# != 2 ]]; then
+    echo "Usage $0 in_marc_file outfile_prefix"
+    exit 1
+fi
+
+MARC_FILE=$1
+OUTFILE_PREFIX=$2
+
+for bibwiss_type in "WiReLex" "WiBiLex"; do
+    marc_grep $1 'if "TYP"=="'${bibwiss_type}'" extract "856u"' traditional | \
+    awk '{print $2}' | \
+    xargs -I '{}' /bin/bash -c $'
+        interval=$((1 + $RANDOM % 3)); echo "Sleeping ${interval} seconds" >2; sleep ${interval};
+        ./translate_url_multiple "$1" | \
```

Comment:
Shouldn't the extension of translate_url_multiple be .sh if it is a shell script?

Reply:
As explained.